packages/instrumentation/src/ActiveTracingHelper.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Context,
+  context as _context,
+  ROOT_CONTEXT,
+  Span,
+  SpanContext,
+  SpanKind,
+  trace,
+  TraceFlags,
+} from '@opentelemetry/api'
+import { Span as SpanConstructor, Tracer } from '@opentelemetry/sdk-trace-base'
+import { EngineSpanEvent, ExtendedSpanOptions, SpanCallback, TracingHelper } from '@prisma/internals'
+
+// If true, will publish internal spans as well
+const showAllTraces = process.env.PRISMA_SHOW_ALL_TRACES === 'true'
```

Comment:
Does it make sense to get this to be dynamic (ie. not store it in a module constant?)

Reply:
Not sure. If it's not set when the process is started but assigned to `prisma.env` later it will not get propogated to engine - so at best, inlining it would make it half-working: client internal spans will be exported, but engine ones will not.